router/router_system.go

```diff
@@ -46,12 +46,8 @@ func getSystemInformation(c *gin.Context) {
 
 // Returns list of host machine IP addresses
 func getSystemIps(c *gin.Context) {
-	i, err := system.GetSystemIps()
-	if err != nil {
-		middleware.CaptureAndAbort(c, err)
-		return
-	}
-	c.JSON(http.StatusOK, i)
+	cfg := config.Get()
+	c.JSON(http.StatusOK, system.IpAddresses{IpAddresses: cfg.Docker.SystemIps})
```

Comment:
Not the way to do it, let docker try then append to it the ones from the config the ones that docker did not yet found